plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -188,6 +188,10 @@ func NodeRules() []rbacv1.PolicyRule {
 	if utilfeature.DefaultFeatureGate.Enabled(features.ClusterTrustBundle) {
 		nodePolicyRules = append(nodePolicyRules, rbacv1helpers.NewRule("get", "list", "watch").Groups(certificatesGroup).Resources("clustertrustbundles").RuleOrDie())
 	}
+	// Kubelet needs access to ServiceAccounts to support sending service account tokens to the credential provider.
+	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletServiceAccountTokenForCredentialProviders) {
+		nodePolicyRules = append(nodePolicyRules, rbacv1helpers.NewRule(Read...).Groups(legacyGroup).Resources("serviceaccounts").RuleOrDie())
```

Comment:
Don't need `get` IIUC.

```suggestion
		nodePolicyRules = append(nodePolicyRules, rbacv1helpers.NewRule("list", "watch").Groups(legacyGroup).Resources("serviceaccounts").RuleOrDie())
```